go/ct/common/bytes_test.go

```diff
@@ -99,3 +101,42 @@ func TestBytes_InvalidJsonFails(t *testing.T) {
 		})
 	}
 }
+
+func TestBytes_RandomBytesProducesRandom(t *testing.T) {
+	rnd := rand.New()
+	for i := 0; i < 10000; i++ {
+		b1 := RandomBytes(rnd, 100)
+		allZeros := true
+		for _, b := range b1.data {
+			if b != 0 {
+				allZeros = false
+				break
+			}
+		}
+
+		if len(b1.data) > 1 && allZeros {
+			t.Errorf("RandomBytes produced all zeros: %v, len:v %v", b1.data, len(b1.data))
+		}
+	}
+}
+
+func TestBytes_String(t *testing.T) {
+	b := NewBytes([]byte{1, 2, 3})
+	if b.String() != "0x010203" {
+		t.Errorf("unexpected string representation, got %v", b.String())
+	}
+}
+
+func TestBytes_Length(t *testing.T) {
+	b := NewBytes([]byte{1, 2, 3})
```

Comment:
it is called, all the `XData` types in the state are of type `Bytes`, and it's `Lenght` is queried in multiple places. I will add the empty case.

Reply:
if it was called, why was coverage not accounting for it?

Reply:
package coverage did not account for it, because this method was not called in any other test in this package. However it is used in many other packages and tests (serialization_test, for example), but since these calls are outside of the package scope they are not considered for package coverage.